docs/hopr-documentation/docs/developers/tutorial-hello-world.md

```diff
@@ -0,0 +1,224 @@
+---
+id: tutorial-hello-world
+title: HOPR Apps - Hello world
+---
+
+The following is a 5-minutes guide for you to get familiar with the HOPR protocol and start developing apps on top of the
+HOPR network by learning how to send and read messages from one node to another.
+
+## Requirements
+
+### 1. Previous guides
+
+Before getting started, we suggest you get familiar with the following concepts:
+
+- ["Running a local HOPR Cluster"](#)
+- ["Interacting with a HOPR node"](#)
+
+### 2. HOPR Cluster & exported variables
+
+Make sure you have a HOPR cluster available for you to connect. Also, make sure to have the equivalent of at least `2`
+nodes endpoints (both HTTP and WS). Here are the default values for each of these endpoints.
```

Comment:
Also the nodes need to be funded.

Reply:
I was on the impression that the terminology of `HOPR Cluster` implied nodes with funded channels interconnected between each other. I'll make a footnote about this.

Reply:
Included in [8abc8fb](https://github.com/hoprnet/hoprnet/pull/3397/commits/8abc8fb6f94655355f5de7df84739c33cc3e5b8e)